client/i18n/locales/english/intro.json

```
@@ -1738,7 +1738,7 @@
       },
       "review-basic-html": {
         "title": "Basic HTML Review",
-        "intro": ["Review the basic HTML topics."]
+        "intro": ["Before you are quizzed on the HTML knowledge you have gained so far, you first need to review the concepts. Open up this review page to see which concepts you need to review."]
```

```suggestion
        "intro": [
          "Before you are quizzed on the HTML knowledge you have gained so far, you first need to review the concepts. Open up this review page to see which concepts you need to review."
        ]
```
This should make prettier happy.